src/rebar_cmd.app.src

```diff
@@ -1,8 +1,8 @@
 {application, rebar_cmd,
- [{description, "Run custom shell commands as rebar3 tasks"},
-  {vsn, "0.2.6"},
+ [{description, "Run custom shell commands with rebar3 cmd <command>"},
```

Comment:
Do you like it like this? (I also updated the `.erl`); might be reverted if you don't.